bin/blocks-plot

```diff
@@ -34,14 +34,14 @@ def plot_dataframe(dataframe):
     print("Plotting {} channels:".format(len(dataframe.columns)))
     for cname, series in iteritems(dataframe):
         print("    {}".format(cname))
-        pylab.plot(t, series.interpolate(), label=cname)
```

Comment:
There will be gaps in the  lineplot where the NaNs are. Which is fine, as long as long as you have more than one successive valid datapoint (E.g.: an alternating Series with [1., Nan, 2., NaN, 3., NaN, ...] would be the worse case and not display anything)

But using interpolate will extend all shorter graphs all the way to the right using a straight line.